packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -59,6 +59,11 @@ export class WaferMap extends FoundationElement {
      */
     public readonly canvas!: HTMLCanvasElement;
 
+    /**
+     * @internal
+     */
+    @observable public canvasSide: number | undefined;
```

Comment:
How about `canvasSideLength`? When I first read `canvasSide` I thought it might be an enum (left side, right side, etc) so was a bit confused until I saw its use.

Reply:
applied